client_plugin/drivers/shared/shared_driver.go

```diff
@@ -62,6 +86,18 @@ func NewVolumeDriver(cfg config.Config, mountDir string) *VolumeDriver {
 	}
 	d.dockerd = cli
 
+	// Pull Samba image
+	log.Infof("Pulling Samba server image")
```

Comment:
So if we use the same dockerd to load the image, it will be loaded into to the host's dockerd.
The same as what you have done here. But `ImagePull` -> `ImageLoad`.